src/core/errorHandler.js

```diff
@@ -38,7 +38,17 @@ export class ErrorHandler {
     return (
       <ul className="ErrorHandler-list">
         {this.capturedError.messages.map(
-          (msg) => <li className="ErrorHandler-list-item">{msg}</li>
+          (msg) => {
+            let msgString = msg;
+            if (typeof msgString === 'object') {
+              // This is an unlikely scenario where an API response
```

Comment:
There's no issue and it probably won't happen because I don't know of an API response that does this but it is possible. If it does happen, someone from QA will see a weird JSON object and bug us. But without my protective code, React will throw an error saying that the child node can't be an object.